frodo.js

```diff
@@ -1,41 +1,39 @@
 /**
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.